code/go/0chain.net/smartcontract/storagesc/handler.go

```diff
@@ -415,7 +417,34 @@ func (ssc *StorageSmartContract) GetReadMarkersHandler(ctx context.Context,
 		query.AuthTicket = authTicket
 	}
 
-	readMarkers, err := balances.GetEventDB().GetReadMarkersFromQuery(query)
+	if offsetString != "" {
+		o, err := strconv.Atoi(offsetString)
+		if err != nil {
+			return nil, errors.New("offset is invalid")
+		}
+		offset = o
+	}
+
+	if limitString != "" {
+		l, err := strconv.Atoi(limitString)
+		if err != nil {
+			return nil, errors.New("limit is invalid")
+		}
+		limit = l
+	}
+
+	if sortString != "" {
+		switch sortString {
+		case "desc":
+			isDescending = true
```

Comment:
if you have to use this, change to desc

Reply:
isDescending is a boolean becomes obvious due to the name this is a clean code practice.
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig